### PR TITLE
Mapper413

### PR DESCRIPTION
**Pull request by negativeExponent**
- Add misc roms handling
- Adds mapper 413

Fix https://github.com/TASEmulators/fceux/issues/280
Fix https://github.com/TASEmulators/fceux/issues/269

@ClusterM @zeromus

### CODE REVIEW

**Comment by hossbags2** [comment]
previous build was working. will this be added again

**Comment by zeromus** [comment]
huh?

**Comment by thor2016** [comment]
The builds you are referencing are on different branches. Build 1.0.1755 is a build test for this pull request. Build 1.0.1756 was off of master branch that does not yet have these changes (since the pull request has not yet been merged to master).

File: src/boards/413.cpp [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+	laddr = 0;
+	lreset = 0;
+
+	Sync();
+	
+	SetReadHandler(0x4800, 0x4FFF, M413ReadPCM);
+	SetReadHandler(0x5000, 0x7FFF, CartBR);
+	SetReadHandler(0x8000, 0xBFFF, CartBR);
+    SetReadHandler(0xC000, 0xCFFF, M413ReadPCM);
```
Comment (thor2016) [inline src/boards/413.cpp:121]: The indentation is not consistent here.

Comment (negativeExponent) [reply src/boards/413.cpp]: fixed

**Pull request merged**